### DISCUSSION TITLE
Cygnett devices Australia

### DISCUSSION THREAD

**Discussion by limbotia** [Koenkk/zigbee2mqtt#12455] [Devices]
Hope this is the correct mechanism but confirming the Australian manufacturer Cygnett devices are recognized as standard Sonoff devices.  Have the Smart Button, Motion Detector and Temperature\Humidity sensor all recognized as Sonoff equivalents (SNZB-01/02/03)  working.

**Comment by sam43434** [comment]
did you have to do anything special to get the smart button and motion detector to work,  i can get the Temperature sensor to work but not the other ones

**Comment by limbotia** [comment]
No nothing special.  Just put into pairing mode as was recognised by both
ZHA and Z2MQTT.

**Comment by joelegg** [comment]
I have the cygnettCY2963CHCSA motion sensor but it's failing to be recognised. I am using the new sonof dongle e tho. But every other brand works tho I see cygnett is not on supported list.

**Comment by limbotia** [comment]
I have had up to 6 of the buttons and  two temp/hum sensors all connected via  a Sonoff dongle ( the "old" P version not the new "E"one ) and ZHA and they are just appearing as Sonoff devices. [See attached (only 3 active buttons at the moment) ](https://photos.app.goo.gl/8B7EA6VXQ55eZKBx6). Also had working under Z2MQTT previously.  I'll drag out my motion detector and test that again as it's not currently in service when I get a chance and report back.

**Reply by joelegg** [reply]
were you able to test this?
